target/scripts/startup/fixes-stack.sh

```diff
@@ -35,21 +33,18 @@ function _fix_var_amavis_permissions
   [[ ${ONE_DIR} -eq 0 ]] && AMAVIS_STATE_DIR="/var/lib/amavis"
   [[ ! -e ${AMAVIS_STATE_DIR} ]] && return 0
 
-  _notify 'inf' 'Checking and fixing Amavis permissions'
-  chown -hR amavis:amavis "${AMAVIS_STATE_DIR}"
-
-  return 0
+  _notify 'inf' 'Fixing Amavis permissions'
+  chown -hR amavis:amavis "${AMAVIS_STATE_DIR}" || _shutdown 'Failed to fix Amavis permissions'
 }
 
 function _fix_cleanup_clamav
 {
-  _notify 'task' 'Cleaning up disabled Clamav'
-  rm -f /etc/logrotate.d/clamav-*
-  rm -f /etc/cron.d/clamav-freshclam
+  _notify 'task' 'Cleaning up disabled ClamAV'
+  rm /etc/logrotate.d/clamav-* /etc/cron.d/clamav-freshclam || _shutdown 'Failed to remove ClamAV configuration'
 }
 
 function _fix_cleanup_spamassassin
 {
   _notify 'task' 'Cleaning up disabled SpamAssassin'
-  rm -f /etc/cron.daily/spamassassin
+  rm /etc/cron.daily/spamassassin || _shutdown 'Failed to remove SpamAssassin configuration'
```

Comment:
Is there a reason for dropping `-f` on `rm` calls?

[`rm` docs](https://man7.org/linux/man-pages/man1/rm.1.html):

> `-f`, `--force`
> 
> ignore nonexistent files and arguments, never prompt

I understand that usually `docker-mailserver` isn't run with an interactive terminal, and prompting probably doesn't happen in that case, but is it desirable to be prompted when using something like `docker run -it`? We usually only do `docker run -t` if not using compose commands.

Without `-f` that also implies it'll error if the file does not exist, which in this context doesn't seem like it should be a concern to throw such an error?

Reply:
When using `-f`, `rm` will never fail (exit 1), which makes testing for success pointless.

Reply:
> it'll error if the file does not exist

This should never be the case when using the images we provide?

Reply:
`rm <file>` never prompts for confirmation. However, many Linux distributions ship with some default aliases like:

```sh
alias rm='rm -i' # prompt before every removal
alias cp='cp -i' # prompt before overwrite
```